### PR TITLE
Rover: astyle, reformatting few files in Rover Dir

### PR DESCRIPTION
**Pull request by jbteja**
This PR to re-format few files in Rover Dir _(files btw a & c)_.

+Process followed:+
- Used https://ardupilot.org/dev/docs/style-guide.html for style reference.
- Ran the files with **astyle** tool and Added the changes **manually** which are only appropriate.
- For easier verification and merging, Splitting my changes into **multiple** PR's.

Queries, 
- Shouldn't the function definition's **brace** be-the-same-line ?
- I went thought some of the files and For most func def opening brace is no the next line. 
If it is acceptable, I would update the function braces as well.

PS: New here, and not sure from where to start contributing. 
Any suggestion or references are welcomed :).

### CODE REVIEW

**Comment by khancyr** [comment]
Hello,

Thanks for contributing.
Unfortunately, large size formating of our code isn't something we really want as it doesn't bring much benefit against breaking the file history.
Unless you are fixing important issue in the code style, it would be better to bring style changes when modifying the related code for something else.
Even if we try to keep a unified codestyle, we don't have a strong policy to enforce it

**Comment by peterbarker** [comment]
Further to @khancyr 's comments, some of the changes made here are actually *not* good, even though astyle thinks they are.

git history is more important to me than removing trailing whitespace :-)

**Comment by peterbarker** [comment]
@jbteja we have a list of "good first issues" in github: https://github.com/ardupilot/ardupilot/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22

We have real problems that need to be investigated.  If you feel like tackling something in there, grab it!

**Comment by jbteja** [comment]
Well said. I haven't thought of git history.

Most of the changes here are w.r.t switch, case indentation. Will revert those changes.

**Comment by jbteja** [comment]
@peterbarker, 
Thanks for the response. 
I looked into the issues and there is a learning curve involved..., Trying to understand :-!.

File: Rover/AP_Rally.h [hunk 1]
```diff
@@ -17,8 +17,7 @@
 #include <AP_Rally/AP_Rally.h>
 #include <AP_AHRS/AP_AHRS.h>
 
-class AP_Rally_Rover : public AP_Rally
-{
+class AP_Rally_Rover : public AP_Rally {
```
Comment (jbteja) [inline Rover/AP_Rally.h:20]: @peterbarker , @khancyr ,

Please confirm, If you want me to revert braces related changes in .h files as well.

File: Rover/AP_Arming.h [hunk 2]
```diff
@@ -6,10 +6,8 @@
 /*
   a rover-specific arming class
  */
-class AP_Arming_Rover : public AP_Arming
-{
+class AP_Arming_Rover : public AP_Arming {
```
Comment (jbteja) [inline Rover/AP_Arming.h:9]: @peterbarker , @khancyr ,

Please confirm, If you want me to revert braces related changes in .h files as well.

Comment (jbteja) [reply Rover/AP_Arming.h:9]: I went through some more files/code and Think, there are too many .h, header file we can change.

Reverting my header file changes as well. If we decided to add, Ill raise a separate PR in the feature.

**Comment by peterbarker** [comment]
@jbteja I think I recognise that arming change - the indenting is wrong but I believe that was done to explicitly preserve history on those lines.

I can't emphasise enough that unless the formatting is ridiculously bad it's more important to preserve history than make everything look pretty.  There's also the time taken to review the PRs and put them through the process.  Much better IMO to take a real issue and deal with it - remember that we *know* the astyle tool is there and we do know how to use it, too :-)  The sorts of changes made in here would have made my tracing of the introduction of a bug into the code on the weekend  harder, for example.

This is out list of good-first-issues: https://github.com/ArduPilot/ardupilot/labels/good%20first%20issue - there's plenty of things in there that will help you get to know the codebase better than making codestyle changes :-)

All that being said, Randy has previously merged this sort of PR, so if you really want to push this through it could probably be done.

**Comment by jbteja** [comment]
I reverted most of the changes from yesterday considering the same point and limited the changes to just spaces & \n. 

Expect from a extra comma in Rover/AP_Arming.cpp, Nothing else is changed. 
I don't think it would brick the code :), Anyways Thanks.

**Comment by jbteja** [comment]
And, I went though astyle doc and Guess, we can add couple of options to astylerc file to align with the projects code style.
-  **keep-one-line-blocks**  <- Don't break one-line blocks, Which we follow.
- **pad-comma**  <- Insert space padding after commas

Change the style flag to attach form linux:
- **style=attach**  <- This option will keep braces on the same line of func, class.

Would raise a  PR if its worth raising/interested for the team. Please comment. Thanks.

**Comment by rmackay9** [comment]
@jbteja, this looks good, thanks.  If you can squash the commits down to a single commit I'll merge it.

**Comment by jbteja** [comment]
Hi @rmackay9 ,
Squash and pushed the changes into a single commit. Thanks.

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request merged**